bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
I'm happy to forbid this, but on a brief experiment (I just renamed all my package directories to have a space in), everything seems to work fine. We could at some point add an explicit check to reject such packages from `LoadPackage` (or somewhere similar).